rfcs/sliders-component.md

```diff
@@ -0,0 +1,122 @@
+# Sliders Component
+
+Represents control with sliding axis to choose a single value or range between min and max values.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {Sliders} from 'baseui/sliders';
+
+export default () => <Sliders range={[
+  {value: 0, label: '0, min'},
+  {value: 100, label: '100, max'}
+]} />;
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {Foo} from 'baseui/foo';
+
+export default () => {
+  return <Sliders
+    range={[
+      {value: 0, label: '0, min'},
+      {value: 20, label: '20'},
+      {value: 40, label: '40'},
+      {value: 60, label: '60'},
+      {value: 80, label: '80'},
+      {value: 100, label: '100, max'}
+    ]}
+    thumbs={4}
+    initialState={{
+      selectedValues: [
+        {value: 20, label: '20'}, {value: 40, label: '40'},
+        {value: 60, label: '60'}, {value: 80, label: '80'},
+      ]
+    }}
+    step={20}
+    onChange={(event, selectedValues)=>{console.log(selectedValues)}}
+    overrides={{
+      Thumb: props => <div style={{
+        color: 'red',
+        borderRadius: '5px',
+      }}></div>,
+    }}
+    />
+}
+```
+
+## Exports
+
+* `Sliders`
+* `StatefulSliders`
+* `StatefulSlidersContainer`
+* `StyledRoot`
+* `StyledAxis`
+* `StyledTick`
+* `StyledThumb`
+* `CONSTANT`
+
+## `Sliders` API
+
+* `range: {Array<Point>}` - Required
+  All points present on slider axis. First and last point represent min and max values. Each `Point` can be a value itself (for primitive type) or if `Point` is `Object` it should have `value` and `label`. If more than 2 elements are present in array, they represent all ticks on axis and `step` property is ignored.
+* `thumbs: number` - Optional
+  Amount of thumbs representing how many values can be selected. For even quantity of thumbs selected range is considered in between them. Default is `1`
+* `selectedValues: {Array<Point>}` - Optional
+  Selected array of points from left to right chosen on axis. Default is `[]`
```

Comment:
Thoughts on changing this to `value`, more like the `<input type="range" />`?

Also, what do you think about supporting `Point | Array<Point>`.  `<Slider value={5}/>` feels a lot more natural for the common single thumb use case.

Reply:
yes, I can do it, but user need to check if its array or just a value, if amount of thumbs is a variable
renamed to `values`